.babelrc

```diff
@@ -0,0 +1,5 @@
+{
+    "presets": [
+        "@babel/preset-env"
+    ]
+ }
```

Comment:
New line needed here

Reply:
Please can i correct this on Monday?
I do not have my laptop here to effect this change.

Reply:
sure it is not immediate